mathjax3-ts/output/chtml/Wrappers/msubsup.ts

```diff
@@ -162,8 +171,9 @@ export class CHTMLmsubsup extends CHTMLscriptbase {
         if (this.UVQ) return this.UVQ;
         const tex = this.font.params;
         const t = 3 * tex.rule_thickness;
-        let [u, v] = (this.isCharBase() ? [0, 0] : [this.getU(basebox, supbox),
-                       Math.max(basebox.d + tex.sub_drop * subbox.rscale, tex.sub2)]);
+        const subscriptshift = this.length2em(this.node.attributes.get('subscriptshift'), tex.sub2);
+        const drop = (this.isCharBase() ? 0 : basebox.d + tex.sub_drop * subbox.rscale);
```

Comment:
These (also below) seem important computations. Can you add a comment what exactly goes on for the benefit of others (e.g, this reviewer).

Reply:
These are the computations given in the TeXbook (Appendix G).  I've cited that in the comment for the function itself.  It's not always clear to me what they are up to, either, as the comments in the TeXBook are pretty sparse as well.  I'll see what I can do.